app/assets/stylesheets/darkswarm/_shop-modals.css.scss

```diff
@@ -1,9 +1,30 @@
 .product-header {
+  padding-left: 1.5rem;
+
   h1, h2, h3, h4, h5, h6 {
-    margin: 0;
+    margin: 0.2rem 0 0;
+  }
+
+  span {
+    color: $grey-500;
   }
 
-  hr {
-    margin: 0.5em 0;
+  .product-description {
+    margin: 1rem 0.25rem 0.25rem 0;
+  }
+
+  .property-selectors li {
+    margin: 0 0.25rem 0.25rem 0;
+    padding: 0.4rem 0 0;
+
+    a {
+      border: 1px solid $grey-300;
+      font-weight: normal;
+      padding: 0.2em 0.625em;
```

Comment:
what not `rem` here? won't make it harder to change if we don't stick to `rem` as we do in the rest of the file?

Reply:
I think I copied this from the shop filters. I changed it now to rem.